examples_tests/CMakeLists.txt

```diff
@@ -18,15 +18,14 @@ add_subdirectory(03.GPU_Mesh EXCLUDE_FROM_ALL)
 add_subdirectory(04.BuiltinNodes EXCLUDE_FROM_ALL)
 add_subdirectory(05.SpherePoints EXCLUDE_FROM_ALL)
 add_subdirectory(06.MeshLoaders EXCLUDE_FROM_ALL)
-add_subdirectory(07.HardwareSkinning EXCLUDE_FROM_ALL)
+if (IRR_BUILD_CEGUI)
+add_subdirectory(07.BRDF_Explorer EXCLUDE_FROM_ALL)
+endif()
 add_subdirectory(08.HardwareInstancing EXCLUDE_FROM_ALL)
 add_subdirectory(09.ColorSpaceTest EXCLUDE_FROM_ALL)
 add_subdirectory(10.AllocPerfTest EXCLUDE_FROM_ALL)
 add_subdirectory(11.RayCastCollision EXCLUDE_FROM_ALL)
 add_subdirectory(12.Splines EXCLUDE_FROM_ALL)
-if (IRR_BUILD_CEGUI)
-add_subdirectory(13.BRDF_Explorer EXCLUDE_FROM_ALL)
-endif()
```

Comment:
NO, this is not what I asked for AT ALL

Example 13 stays as the BRDF Explorer

Example 7 is supposed to be a pure CEGUI-only example to show off GUI, nothing else